### PR TITLE
Maximize consecutive average integer pair in C++

### PR DESCRIPTION
**Pull request by sneha-nitdgp**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes.)

### Description

(A pair of two consecutive members u and v on a line is considered average integer pair if their average is an integer,
i.e. (u+v)/2 is an integer. Given an array, arrange the elements, such that number of consecutive average integer pairs is maximised.
I've added C++ solution to the above problem in others section of C plus plus.)

### Checklist

- [✔️  ] I've read the contribution guidelines.
- [ ✔️ ] I've checked the issue list before deciding what to submit.
- [✔️  ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

[#5986](https://github.com/TesseractCoding/NeoAlgo/issues/5986)

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by sneha-nitdgp** [comment]
@ankitaggarwal23 Updated! Please check.

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -599,4 +599,5 @@ _add list here_
 - [Repeating and Missing Number](other/repeated_missing.cpp)
 - [Kth missing Positive integer in sorted Array](other/kth_missing.cpp)
 - [Triplet Sum in array](other/Triplet_sum_in_array.cpp)
+- [Maximise consecutive average integer pair](other/Maxm_consecutive_avgint_pairs.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Typo fix `Maximise`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Code produces error check braces and their alignment

File: C-Plus-Plus/other/Maxm_consecutive_avgint_pairs.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Input:
+Enter the no of elements in array:
+8
+Enter the elements in array:
+10 9 13 15 3 16 9 13
+Output:
+The reordered array is: 
+9 13 15 3 9 13 10 16 
+
+Tme complexity:-O(n) to compute odd and even numbers
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/Maxm_consecutive_avgint_pairs.cpp]: Fix the typo

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/other/Maxm_consecutive_avgint_pairs.cpp (hunk 2 above)
Comment (sneha-nitdgp) [reply C-Plus-Plus/other/Maxm_consecutive_avgint_pairs.cpp]: Which typo? I had fixed the one you had mentioned earlier.

Comment (ankitaggarwal23) [reply C-Plus-Plus/other/Maxm_consecutive_avgint_pairs.cpp]: Time not tme

Comment (sneha-nitdgp) [reply C-Plus-Plus/other/Maxm_consecutive_avgint_pairs.cpp]: Had overlooked that, fixed now.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**